src/Spaces/pointspace.jl

```diff
@@ -17,17 +17,26 @@ ClimaComms.device(space::PointSpace) = ClimaComms.device(space.context)
 ClimaComms.context(space::PointSpace) = space.context
 
 @deprecate PointSpace(x::Geometry.LocalGeometry) PointSpace(
-    ClimaComms.SingletonCommsContext(ClimaComms.CPUDevice()),
+    ClimaComms.CPUDevice(),
     x,
 ) false
+@deprecate PointSpace(x::Geometry.AbstractPoint) PointSpace(
+    ClimaComms.CPUDevice(),
+    x,
+) false
```

Comment:
I'm still a bit reluctant to deprecate these directly...

Reply:
Sure, I think that’s fine. I’ll update

Reply:
Updated